chatmaild/src/chatmaild/config.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, inipath, params):
         self.passthrough_recipients = params["passthrough_recipients"].split()
         self.filtermail_smtp_port = int(params["filtermail_smtp_port"])
         self.postfix_reinject_port = int(params["postfix_reinject_port"])
-        self.disable_ipv6 = params.get("disable_ipv6")
+        self.disable_ipv6 = True if params.get("disable_ipv6") == "True" else False
```

Comment:
```suggestion
        self.disable_ipv6 = True if params.get("disable_ipv6").lower() == "true" else False
```

On second thought, we can't expect admins to case their variables correctly.